### ISSUE TITLE
Project Presentation shit

### ISSUE THREAD

**Issue by R31ZH** [R31ZH/Project-1#28]

**Comment by bighamcyrus** [comment]
for presentation:

 Cyrus will present the code for the image generator and what challengessolutions he faces with it. Will also present the code for the download button and challenges/solutions faced. 

 Jeremiah - will show us how he got the random quotes to generate as well as how he created the share button. What challenges/solutions did you find will doing it. 

 Jenny - will show us what she had to do to change the colors and text of the page. What challenges did you come up with when learning a new CSS template like tailwinds 

 Maximus - will present us with what he did with the boxes and buttons. What challenges did you come find when learning the new css styling/solutions.

**Comment by ivetteramos** [comment]
Jenny: I created the framework HTML using TailWind. I had to create two spaces where on the right we added the search input for images. The left side was added to have a space where the image will generate. Using TailWind for the first time was a bit of a struggle trying to read and change their fonts and colors. At the end it was nice to be able to add just one class for all the hover, colors, borders, and flex syntax.
